src/trans/cpu/internal/ledir_mod.F90

```diff
@@ -250,7 +250,10 @@ SUBROUTINE LEDIR(KM,KMLOC,KFC,KIFC,KSL,KDGLU,KLED2,PAIA,PSIA,POA1,PW)
       POA1(IS+(J-1)*2,JK) = ZCS(J,IFLD)
     ENDDO
   ENDDO
-  
+ELSE
+  ! This zonal wavenumber KM has no computation to be done, probably because the wavenumber cannot
```

Comment:
It is specifically because G%NDGLU(KM) = 0 that we are not entering the IF(), but guess this isn't really saying much more than the inline comment you've left.